### ISSUE TITLE
New experiment: riseupvpn

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe-engine#985]
This issue describes the work I will perform to provide feedback to @cyBerta for https://github.com/ooni/probe-engine/pull/978 and to @KwadroNaut for https://github.com/ooni/spec/pull/202. It's not possible to predict whether merging is possible before end of Sprint, but we should certainly do our best to provide feedback and enable progress.

**Comment by bassosimone** [comment]
Moving to Sprint 25. In Sprint 24 we managed to do the following:

1. review and merge the test spec, provide some design questions to think about (in https://github.com/ooni/spec/pull/202)

2. review ~50% of the code PR, provide suggestions (in https://github.com/ooni/probe-engine/pull/978)

In both cases I tried to provide not just code review but also insights on the overall design and direction. As we discussed recently into an internal @ooni/ooni-team meeting, we want to ensure we're broadening the scope of our code reviews.
